save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/DemoManagerController.kt

```diff
@@ -148,23 +165,39 @@ class DemoManagerController(
         description = "Get demo status.",
     )
     @ApiResponse(responseCode = "200", description = "Successfully fetched demo status.")
+    @ApiResponse(responseCode = "403", description = "Not enough permission for accessing given project.")
+    @ApiResponse(responseCode = "404", description = "Could not find project in organization.")
```

Comment:
`Could not find project in organization or demo for it.`

Reply:
Done in [4a36dc6](https://github.com/saveourtool/save-cloud/pull/1688/commits/4a36dc64bb2f0c58be27a075e672c98364c1daac)